CHANGELOG.md

```diff
@@ -1,6 +1,115 @@
+* Add attributes `href`, `vapp_template_list`, `media_item_list`, and `publishing_url` to `vcd_catalog` resource and data source to show published items ([#916](https://github.com/vmware/terraform-provider-vcd/pull/916))
+* Add `subscribed_catalog` to examples ([#916](https://github.com/vmware/terraform-provider-vcd/pull/916))
+* Upgrade Terraform SDK dependency to v2.24.1 ([#920](https://github.com/vmware/terraform-provider-vcd/pull/920), [#930](https://github.com/vmware/terraform-provider-vcd/pull/930))
+* Data source `vcd_nsxt_edge_cluster` deprecated `vdc` field in favor of 3 new fields to define
+  NSX-T Edge Cluster lookup scope - `vdc_id`, `vdc_group_id`, and `provider_vdc_id` ([#921](https://github.com/vmware/terraform-provider-vcd/pull/921))
+* Resource and data source `vcd_org_vdc` introduced new field `edge_cluster_id` to specify NSX-T
+  Edge Cluster for VDC ([#921](https://github.com/vmware/terraform-provider-vcd/pull/921))
+* Resources, that are removed outside of Terraform control are removed from state and recreated
+  instead of returning error ([#925](https://github.com/vmware/terraform-provider-vcd/pull/925))
+  * `vcd_edgegateway_settings`
+  * `vcd_vapp_network`
+  * `vcd_vm_internal_disk`
+  * `vcd_nsxv_dhcp_relay`
+  * `vcd_vapp_static_routing`
+  * `vcd_vapp_nat_rules`
+  * `vcd_vapp_firewall_rules`
+  * `vcd_vapp_access_control`
+  * `vcd_nsxt_alb_edgegateway_service_engine_group`
+  * `vcd_org_vdc`
+  * `vcd_org_user`
+  * `vcd_external_network`
+* Resource and data source `vcd_nsxt_network_dhcp` support Isolated networks, different DHCP modes
+  ('EDGE', 'NETWORK', 'RELAY') and lease time ([#929](https://github.com/vmware/terraform-provider-vcd/pull/929))
+* Add the new attributes `vapp_template_id`, `boot_image_id` to the resources `vcd_vapp_vm` and `vcd_vm` to be able
+  to use unique URNs to reference vApp Templates and Media items through data sources, to build strong dependencies
+  in Terraform configuration. ([#931](https://github.com/vmware/terraform-provider-vcd/pull/931))
+
+## BUG FIXES
+* Fix bug where VM was power cycled multiple times during creation ([#901](https://github.com/vmware/terraform-provider-vcd/pull/901))
+* Fix bug where storage_profile is ignored for empty (non template) VM ([#901](https://github.com/vmware/terraform-provider-vcd/pull/901))
+* `resource/vcd_nsxt_alb_edgegateway_service_engine_group` field `reserved_virtual_services` accepts
+  "0" as value ([#923](https://github.com/vmware/terraform-provider-vcd/pull/923))
+* Fixed a bug in `resource/vcd_vapp` that would prevent to Power off vApp when previous state was
+  `power_on=true` ([#932](https://github.com/vmware/terraform-provider-vcd/pull/932))
+
+## DEPRECATIONS
+Deprecate `vcd_external_network` in favor of `vcd_external_network_v2` ([#903](https://github.com/vmware/terraform-provider-vcd/pull/903))
+* Deprecated `default_vm_sizing_policy_id` field in `vcd_org_vdc` resource and data source. This field is misleading as it
```

Comment:
As a general comment, probably we should align all verb tenses.

Reply:
Done